src/utils/emails.py

```diff
@@ -195,6 +195,30 @@ def get_email_context(customer=None, target=None, url=None):
     }
 
 
+def clean_from_address(template_from_address: str):
+    """Clean processed from address."""
+    # Get template display name
+    if "<" in template_from_address:
+        template_display = template_from_address.split("<")[0].strip()
+        template_sender = template_from_address.split("@")[0].split("<")[-1]
+        sp_domain = template_from_address.split("@")[1].split(">")[0]
+    else:
+        template_display = ""
+        template_sender = template_from_address.split("@")[0]
+        sp_domain = template_from_address.split("@")[1]
```

Comment:
this works, but using regular expressions is a better solution

Reply:
@xlyk check this out: [5dee7d9](https://github.com/cisagov/con-pca-api/pull/593/commits/5dee7d9cfe171dc81b8ffc1e1a6f779e0c1e2499)